v_4_4_0/types.go

```diff
@@ -97,8 +97,18 @@ type FileMetadata struct {
 }
 
 type Owner struct {
-	User  string
-	Group string
+	Group Group
+	User  User
+}
+
+type Group struct {
+	ID   int
+	Name string
+}
+
+type User struct {
```

Comment:
I'd like to have a godoc that ID and Name are mutually exclusive and if both are set then ID is preferred.